spec/pluralrules.html

```diff
@@ -0,0 +1,360 @@
+<emu-clause id="pluralrules-objects">
+  <h1>PluralRules Objects</h1>
+
+  <emu-clause id="sec-intl-pluralrules-abstracts">
+    <h1>Abstract Operations for PluralRules Objects</h1>
+
+    <emu-clause id="sec-initializepluralrules" aoid="InitializePluralRules">
+      <h1>InitializePluralRules ( _pluralRules_, _locales_, _options_ )</h1>
+
+      <p>
+        The abstract operation InitializePluralRules accepts the arguments _pluralRules_ (which must be an object), _locales_, and _options_. It initializes _pluralRules_ as a PluralRules object. The following steps are taken
+      </p>
+
+      <emu-alg>
+        1. Let _requestedLocales_ be ? CanonicalizeLocaleList(_locales_).
+        1. If _options_ is *undefined*, then
+          1. Let _options_ be ObjectCreate(*null*).
```

Comment:
this is the only point of contention as far as I can tell with respect to what was approved for stage 4 few weeks ago.

Reply:
Added a test at https://github.com/tc39/test262/pull/1350 . This should be all good.